app/models/server/raw/Rooms.js

```diff
@@ -207,6 +207,7 @@ export class RoomsRaw extends BaseRaw {
 			$unset: {
 				teamId: '',
 				teamDefault: '',
+				teamMain: '',
```

Comment:
`unsetTeamIdOfRooms` is only called by `teams.delete` and now by the new `teams.convertToChannel` endpoint. This change just assures that none of the team's rooms are going to remain as its main channel after it is deleted or converted to a channel.

So in practical terms, when a team is being **deleted** this new line isn't going to have any effects (since the teamMain must be deleted within the team). And when a team is being **converted** to a channel, it just assures that its main channel is not going to remain as the teamMain of a nonexistent team.

Then I think this wouldn't affect anything else but these two endpoints 🙂